app/services/mentors/create.rb

```diff
@@ -25,10 +25,10 @@ def call
           participant_identity: Identity::Create.call(user: user),
         }.merge(mentor_attributes)) do |mentor_profile|
           ParticipantProfileState.create!(participant_profile: mentor_profile)
-          ParticipantMailer.participant_added(participant_profile: mentor_profile).deliver_later
+          # ParticipantMailer.participant_added(participant_profile: mentor_profile).deliver_later
```

Comment:
https://github.com/DFE-Digital/early-careers-framework/pull/1695/files#r799591130

Reply:
we probably want to uncomment before we merge. if you want to disable these can we not wrap around a feature toggle?

Reply:
https://github.com/DFE-Digital/early-careers-framework/pull/1695#discussion_r800552898

I'll revert this. We need to fix this, in a separate PR any. But without those I won't be able to load the seeds, as it spams sidekiq